Detectors/ITSMFT/common/reconstruction/src/DecodingStat.cxx

```diff
@@ -24,52 +24,31 @@ constexpr std::array<std::string_view, ChipStat::NErrorsDefined> ChipStat::ErrNa
 uint32_t ChipStat::getNErrors() const
 {
   uint32_t nerr = 0;
-  for (int i = NErrorsDefined; i--;) {
-    nerr += errorCounts[i];
-  }
+  auto inner = [&](int errIdx) {
+    nerr += errorCounts[errIdx];
+  };
+  ChipStat::forEachError(inner);
   return nerr;
 }
 
 ///_________________________________________________________________
 /// print link decoding statistics
-uint32_t ChipStat::addErrors(uint32_t mask, uint16_t chID, int verbosity)
+void ChipStat::addErrors(const ChipPixelData& d, int verbosity)
```

Comment:
Why do you suppress return value? Before it is used to decide if the corrupted raw data should be flagged. Latter accounting of the return value was lost https://github.com/AliceO2Group/AliceO2/commit/3cbcf82dfe76f8f3766797aae96eb8228f3f18a5#diff-ac9fa848ea4454ebf49302066cf221333b7bb62f172196b7b6a0616f2f0cc5cfR54-R66
I am now trying to clarify with @freidt if I did this on his demand (to not produce raw data dumps) or if it was a simple omission, but I would keep the return value.

Reply:
Neglecting the return value was indeed an accidental omission, fixed in https://github.com/AliceO2Group/AliceO2/pull/13267.
Please rebase your PR after I merge mine and restore the return value.